includes/customer-functions.php

```diff
@@ -636,14 +636,14 @@ function edd_maybe_add_customer_address( $customer_id = 0, $data = array() ) {
 		'region'      => '',
 		'country'     => '',
 		'postal_code' => '',
-		'type'        => 'billing',
 	);
 	$address_to_check = array_intersect_key( $data, $empty_address );
 	$address_to_check = array_filter( $address_to_check );
 	if ( empty( $address_to_check ) ) {
 		return false;
 	}
 	$address_to_check['customer_id'] = $customer_id;
+	$address_to_check['type']        = 'billing';
```

Comment:
Yes, originally, `$data['type']` was set above, but I've removed that based on a discussion with Ashley. Spent a lot of time thinking about this line, actually, before committing. I ended up intentionally choosing `$address_to_check` over `$data` for two reasons:
1. If an extension wants to set the address `type` to something other than `billing`, setting `$data['type']` here will override that.
2. The default value for the `type` column is `billing`, so if it is not set at all in this process, it will still be `billing` ([ref](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/dbe38863f8d29e171db96f55a95708c1a4d09d84/includes/database/schemas/class-customer-addresses.php#L60)).

Currently testing changing this line to this:

```php
$address_to_check['type'] = empty( $data['type'] ) ? 'billing' : $data['type'];
```

Which now accomplishes the following:
- if a `type` other than `billing` is set, such as `shipping`, it will not be overridden
- if a `billing` address exists and an identical address with a different `type` is passed to the function, it will be added. As long as the `type` label is displayed with the address, this should be clear when viewing, but I would need to double check the customer table PR.

Still testing, but removing line 629 and making this change seems to be effective.